src/command/build.rs

```diff
@@ -133,15 +133,19 @@ impl Build {
         info!(&log, "Done in {}.", &duration);
         info!(
             &log,
-            "Your wasm pkg is ready to publish at {:#?}.", &self.out_dir
+            "Your wasm pkg is ready to publish at {}.",
+            self.out_dir.display()
         );
 
         PBAR.message(&format!("{} Done in {}", emoji::SPARKLE, &duration));
 
         PBAR.message(&format!(
-            "{} Your wasm pkg is ready to publish at {:#?}.",
+            "{} Your wasm pkg is ready to publish at {}.",
             emoji::PACKAGE,
-            self.out_dir.canonicalize().unwrap_or(self.out_dir.clone())
+            self.out_dir
+                .canonicalize()
```

Comment:
I think this may actually be a case where the call to `canonicalize()` could be omitted entirely? This could in that case simply ber `self.out_dir.display()` perhaps?